worlds/mlss/__init__.py

```diff
@@ -90,13 +89,15 @@ def create_regions(self) -> None:
         self.get_location(LocationName.PantsShopStartingFlag1).place_locked_item(item)
         item = self.create_item("Chuckle Bean")
         self.get_location(LocationName.PantsShopStartingFlag2).place_locked_item(item)
+        item = self.create_item("Victory")
```

Comment:
Having a physical, *non-event* Victory item does mean that people can plando it or put it in their starting inventory. I would like you to confirm whether you're ok with that

Reply:
The item was only intended for showing goal in the spoiler log, so it makes sense to make it an event.